data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+<div style={{textAlign: "center",  marginBottom: 10}}>
+**EuroPython 2024 Call for Proposals (CFP) is Open.**
+
+It will be running between
+**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+---
+
+EuroPython reflects the colourful and diverse backgrounds, cultures and
+interests of our community, so you (yes, you!) should go for it: **propose
+something and represent!**
```

Comment:
“and represent”... what?
“propose something and present”?
“propose something to present at EuroPython”?

Reply:
It is a youthful way of encouraging people to be a symbol or to be an outstanding example of whatever they choose. It is deliberately used here as an intransitive verb.

Reply:
Yeah, you can use it even as an expression "Represent!" :)